src/modules/simulator/CMakeLists.txt

```diff
@@ -53,6 +53,8 @@ if (NOT ${OS} STREQUAL "qurt")
 		simulator_mavlink.cpp)
 endif()
 
+add_subdirectory(ledsim)
+
 px4_add_module(
```

Comment:
would be nice to free ecl a from px4 dependencies such that it could be included similar to the simulator